src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -326,70 +333,106 @@ export function openFacilityPageV2(page, uiSchema, schema) {
     try {
       const initialState = getState();
       const newAppointment = initialState.newAppointment;
-      const data = newAppointment.data;
       const typeOfCare = getTypeOfCare(newAppointment.data);
       const typeOfCareId = typeOfCare.id;
       const userSiteIds = selectSystemIds(initialState);
       const useVSP = vaosVSPAppointmentNew(initialState);
+      let typeOfCareFacilities =
+        newAppointment.facilities[typeOfCareId] || null;
+      let siteId = null;
+      let facilityId = newAppointment.data.vaFacility;
+      let parentId = newAppointment.data.vaParent;
       let parentFacilities = newAppointment.parentFacilities;
-      let locations = null;
-      let locationId = data.vaFacility;
+      let directFacilityIds = [];
+      let requestFacilityIds = [];
 
       dispatch({
         type: FORM_PAGE_FACILITY_V2_OPEN,
       });
 
-      try {
-        if (!parentFacilities) {
-          dispatch({ type: FORM_FETCH_PARENT_FACILITIES });
-          parentFacilities = await getOrganizations({
-            siteIds: userSiteIds,
-            useVSP,
-          });
-          dispatch({
-            type: FORM_FETCH_PARENT_FACILITIES_SUCCEEDED,
-            parentFacilities,
+      // Fetch parent facilities if we haven't already in
+      // checkCommunityCareEligibility()
+      if (!parentFacilities) {
+        parentFacilities = await getOrganizations({
+          siteIds: userSiteIds,
+          useVSP,
+        });
+      }
```

Comment:
With community care enabled, we perform this call on the previous page.  But in the case it is disabled, we still fetch `parentFacilities` here because I believe we occasionally need to determine `siteId` throughout the flow, which relies on `parentFacilities`.  There might be a way to exclude this call but it might require refactoring some of the selectors we use so I've left it for now.